nunit.sln

```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```

Comment:
With a bit more seriousness: I have always tried to keep using the lowest version of the IDE that will work for building NUnit. That keeps it working for others who may want to contribute but can't afford Microsoft prices. The VS2015 community edition may change my approach though.